src/machines/onboardingMachine.tsx

```diff
@@ -0,0 +1,260 @@
+import {CredentialPayload} from '@veramo/core';
+import {createContext} from 'react';
+import {v4 as uuidv4} from 'uuid';
+import {assign, createMachine, interpret} from 'xstate';
+import {EMAIL_ADDRESS_VALIDATION_REGEX} from '../@config/constants';
+import {onboardingStateNavigationListener} from '../navigation/onboardingStateNavigation';
+import {walletSetup} from '../services/onboardingService';
+import {SupportedDidMethodEnum} from '../types';
+import {
+  ICreateOnboardingMachineOpts,
+  IInstanceOnboardingMachineOpts,
+  IOnboardingMachineContext,
+  NextEvent,
+  OnboardingContextType,
+  OnboardingEvents,
+  OnboardingEventTypes,
+  OnboardingGuards,
+  OnboardingInterpretType,
+  OnboardingStates,
+  PersonalDataEvent,
+  PinSetEvent,
+  PrivacyPolicyEvent,
+  TermsConditionsEvent,
+} from '../types/onboarding';
```

Comment:
Yeah, I didn't do it in this case because other code should not really handle onboarding. It is such a one time thing. So I didn't want to clutter the global types import with it